portal-api/api-tests/v1/gef/facilities.api-test.js

```diff
@@ -282,6 +282,43 @@ describe(baseUrl, () => {
       expect(status).toEqual(200);
     });
 
+    it('name is required if hasBeenIssued', async () => {
+      const { details } = newFacility;
+      const update = {
+        hasBeenIssued: true,
+        name: null,
+        shouldCoverStartOnSubmission: true,
+        coverStartDate: null,
+        coverEndDate: '2015-01-01T00:00:00.000Z',
+        monthsOfCover: 12,
+        details: ['test'],
+        detailsOther: null,
+        currency: 'GBP',
+        value: 10000000,
+        coverPercentage: 80,
+        interestPercentage: 40,
+        paymentType: 'IN_ADVANCE_QUARTERLY',
+      };
+      const item = await as(aMaker).post({ applicationId: mockApplication.body._id, type: FACILITY_TYPE.CASH, hasBeenIssued: false }).to(baseUrl);
```

Comment:
should `hasBeenIssued` be true here?

Reply:
it's working based on the update so the first post is made to be valid. Doesn't affect the test overall